controllers/admin_context.py

```diff
@@ -0,0 +1,253 @@
+# -*- coding: utf-8 -*-
+import json
+import datetime
```

Comment:
```suggestion
from datetime import datetime as dt
```

Save yourself some headaches, memory, and typing to be more effecient writing and running your code :)

Reply:
Got it!

As soon as everything is merged, ill scratch all of these library imports at once!